vision/beta/cloud-client/src/main/java/com/example/vision/Detect.java

```diff
@@ -0,0 +1,1271 @@
+/**
+ * Copyright 2017, Google, Inc.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
```

Comment:
Please use the same formatting for license as in the other files.
<p> tags don't seem to be required here.

Reply:
Updated